css/style-middle-section.css

```diff
@@ -0,0 +1,173 @@
+@media (min-width: 375px) {
+  #works {
+    /* 375 x 2788 */
+    background-color: #e5e5e5;
+    display: grid;
+    grid-gap: 88px;
+    padding: 114px 24px 114px 24px;
+  }
+
+  .card-work {
+    width: 87.2%;
+    border-radius: 16px;
+    padding: 16px 16px 28px 16px;
+    gap: 12px;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .first-text-section h1 {
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    font-weight: 700;
+  }
+
+  .frame {
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    margin-top: 12px;
+  }
+
+  .frame1 {
+    color: #344563;
+    font-style: normal;
+  }
+
+  .dot {
+    margin: 0 12px 0 12px;
+  }
+
+  .frame2 {
+    color: #7a869a;
+  }
+
+  .frame3 {
+    color: #7a869a;
+  }
+
+  .primary-text {
+    font-size: 15px;
+    line-height: 24px;
+    font-style: normal;
+    font-weight: 400;
+    color: #344563;
+    margin-top: 12px;
+  }
+
+  .tags {
+    margin-top: 12px;
+  }
+
+  .action a {
+    color: #396df2;
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    font-style: normal;
+    display: flex;
+    text-align: center;
+  }
+
+  .skills-li a {
+    display: flex;
+    justify-content: space-between;
+    height: 72px;
+    align-items: center;
+  }
+
+  .tags li a {
+    font-size: 12px;
+    line-height: 16px;
+    font-weight: 500;
+    font-style: normal;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    background-color: #ebebff;
+    border-radius: 8px;
+    padding: 4px 12px 4px 12px;
+    margin-right: 8px;
+  }
+
+  .action {
+    width: 126px;
+    height: 48px;
+    border: 1px solid #396df2;
+    padding: 12px;
+    border-radius: 16px;
+    margin-top: 12px;
+  }
+
+  /* About Myself */
+  #myself {
+    background-color: #fff;
+  }
+
+  #resume {
+    width: 172px;
+    height: 48px;
+    padding: 12px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    margin: 24px 0 24px 24px;
+  }
+
+  #resume a {
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  #skills {
+    margin: 0 24px 0 24px;
+    width: 87.2%;
+    padding-bottom: 114px;
+  }
+
+  #skills ul {
```

Comment:
- [ ] Please push the icons element in the about me section a bit to the right.

- Your design
<img width="186" alt="image" src="https://github.com/BiancaMe/Portfolio-Mobile/assets/41163571/dcf02344-9346-4cdb-8051-e0b45fca08dc">

- Figma design
<img width="196" alt="image" src="https://github.com/BiancaMe/Portfolio-Mobile/assets/41163571/60f18720-b319-44d3-b815-f7e8113e4dd8">